### PR TITLE
[T16-4] Centralised Human Resource System

### PR DESCRIPTION
**Pull request by LimYiSheng**
@zhihong8888 @chuazhenwei @ryanchen2018 @xiiaopanda

**Updates Completed:** 
README.adoc
AboutUs.adoc
UserGuide.adoc
DeveloperGuide.adoc

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,11 +1,7 @@
-= Address Book (Level 4)
+= Centralised Human Resource System (CHRS)
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 
-https://travis-ci.org/nusCS2113-AY1819S1/addressbook-level4[image:https://travis-ci.org/nusCS2113-AY1819S1/addressbook-level4.svg?branch=master[Build Status]]
-https://ci.appveyor.com/project/damithc/addressbook-level4[image:https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true[Build status]]
-https://coveralls.io/github/se-edu/addressbook-level4?branch=master[image:https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master[Coverage Status]]
-https://www.codacy.com/app/damith/addressbook-level4?utm_source=github.com&utm_medium=referral&utm_content=se-edu/addressbook-level4&utm_campaign=Badge_Grade[image:https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a[Codacy Badge]]
-https://gitter.im/se-edu/Lobby[image:https://badges.gitter.im/se-edu/Lobby.svg[Gitter chat]]
+https://travis-ci.org/CS2113-AY1819S1-T16-4/main[image:https://travis-ci.org/nusCS2113-AY1819S1/addressbook-level4.svg?branch=master[Build Status]]
```
Comment (jordancjq) [inline README.adoc]: Good that you've added the Travis badge to your repo. Maybe in future, you could include the others as well. (appveyor, coveralls, codacy)

Comment (LimYiSheng) [reply README.adoc]: Hi Jordan, regarding the Travis-CI of the PR, apparently it shows that it has passed on our organisation repository. Screenshot as attached below: 
image 

The errors flagged out by Travis such as trailing white spaces are not found on our organisation repository files. How can we proceed to resolve this mismatch of Travis-CI?

Comment (zhihong8888) [reply README.adoc]: I see 1 other pull request "Feature1" from others having the same Travis error as us.

Comment (jordancjq) [reply README.adoc]: Hi, thanks for putting up the errors found onto the forum. Hope that might have solved the problem! If there’s still issues, probably you could create a new PR, as mentioned.

Comment (LimYiSheng) [reply README.adoc]: Hi Jordan, we will be closing this PR and proceeding with the creation of a new PR.

**Pull request closed without merging**